projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-list/query-list.component.ts

```diff
@@ -25,50 +26,36 @@ export class QueryListComponent implements OnChanges {
     }
   }
 
-  public sortTable(col: queryListColumn) {
+  public sortTable(col: QueryListColumn) {
     switch (col.displayName) {
-      case 'Queries': {
-        this.sort(col);
-        break;
-      }
-      case 'Last submitted by': {
-        this.sort(col);
-        break;
-      }
-      case 'Last submission date': {
-        this.sortDate(col);
-        break;
-      }
+      case 'Last submission date':
       case 'Last response date': {
         this.sortDate(col);
         break;
       }
-      case 'Response by': {
-        this.sort(col);
-        break;
-      }
       default: {
         this.sort(col);
         break;
       }
     }
   }
 
-  public sortWidget(col: queryListColumn): string {
+  public sortWidget(col: QueryListColumn): string {
     switch (col.sortOrder) {
-      case SortOrder.DESCENDING: {
-        return '&#9660;';
-      }
       case SortOrder.ASCENDING: {
         return '&#9650;';
       }
       default: {
-        return '&#11047;';
+        return '&#9660;';
       }
     }
   }
 
-  private sort(col: queryListColumn) {
+  public showDetails(query) {
```

Comment:
@DavidJayakumar - return types needed for both functions

Reply:
as per Dan's comment